Makefile

```diff
@@ -107,6 +107,8 @@ endif
 
 .PHONY: binaries
 binaries: clean \
+    install-pkgs \
+    generate \
```

Comment:
`make binaries` should not depend on these targets.

These targets should be only called via CI or via a manual invocation

Reply:
Will make generate depend on install-tools. This way its easy to generate proto without running too many targets

Reply:
The problem is that this will clover the command binaries installed by the user.

Reply:
Ah okay got it. So its upto dev/user to run install-tools as well if and when needed